cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -550,9 +546,32 @@ public void changeFlowletStreamConnection(HttpRequest request, HttpResponder res
                                             @PathParam("flow-id") String flowId,
                                             @PathParam("flowlet-id") String flowletId,
                                             @PathParam("stream-id") String streamId) throws IOException {
-    programLifecycleHttpHandler.changeFlowletStreamConnection(RESTMigrationUtils.rewriteV2RequestToV3(request),
-                                                              responder, Constants.DEFAULT_NAMESPACE,
-                                                              appId, flowId, flowletId, streamId);
```

Comment:
this code was moved from ProgramLifecycleHttpHandler, with the default namespace instead of receiving a path parameter. Other than that no change.